srv/salt/ceph/stage/prep/minion/default-no-update-no-reboot.sls

```diff
@@ -1,30 +1,30 @@
 repo:
   salt.state:
-    - tgt: '{{ salt['pillar.get']('deepsea_minions') }}'
+    - tgt: {{ salt['pillar.get']('deepsea_minions') }}
```

Comment:
in QA we are currently using the "default-no-update-no-reboot" alternative default (i.e. this file) and the removal of the single quotes in conjunction with deepsea_minions being set to `'*'` is causing Stage 0 to fail like so:

```
2018-08-23T18:11:45.655 INFO:teuthology.orchestra.run.target167114225022.stdout:    Data failed to compile:
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:----------
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:    Rendering SLS 'base:ceph.stage.0.minion.default' failed: expected alphabetic or numeric character, but found '\n'; line 3
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:---
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:repo:
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:  salt.state:
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:    - tgt: *    <======================
2018-08-23T18:11:45.656 INFO:teuthology.orchestra.run.target167114225022.stdout:    - tgt_type: compound
2018-08-23T18:11:45.657 INFO:teuthology.orchestra.run.target167114225022.stdout:    - sls: ceph.repo
2018-08-23T18:11:45.657 INFO:teuthology.orchestra.run.target167114225022.stdout:
2018-08-23T18:11:45.657 INFO:teuthology.orchestra.run.target167114225022.stdout:metapackage minions:
2018-08-23T18:11:45.657 INFO:teuthology.orchestra.run.target167114225022.stdout:  salt.state:
2018-08-23T18:11:45.657 INFO:teuthology.orchestra.run.target167114225022.stdout:[...]
```

Pushing a commit to #1305 to fix.